packages/@aws-cdk/aws-vpclattice-alpha/lib/service-network.ts

```diff
@@ -0,0 +1,150 @@
+import { CfnServiceNetwork } from 'aws-cdk-lib/aws-vpclattice';
+import { Stack, ArnFormat } from 'aws-cdk-lib/core';
+import { Construct } from 'constructs';
+import { IService } from './service';
+import { ServiceNetworkBase, IServiceNetwork } from './service-network-base';
+
+/**
+ * Authentication types for VPC Lattice service networks
+ */
+export enum ServiceNetworkAuthType {
+  /**
+   * No authentication
+   */
+  NONE = 'NONE',
+
+  /**
+   * AWS IAM authentication
+   */
+  AWS_IAM = 'AWS_IAM',
+}
+
+/**
+ * Properties for creating a VPC Lattice service network
+ */
+export interface ServiceNetworkProps {
+  /**
+   * The name of the service network
+   */
+  readonly serviceNetworkName: string;
+
+  /**
+   * The authentication type for the service network
+   *
+   * @default ServiceNetworkAuthType.NONE
+   */
+  readonly authType?: ServiceNetworkAuthType;
+
+  /**
+   * Whether to enable sharing of the service network
+   *
+   * @default false
+   */
+  readonly enableSharing?: boolean;
+
+  /**
+   * Tags for the service network
+   *
+   * @default - No tags
+   */
+  readonly tags?: string;
+
+  /**
+   * Services to associate with the service network
+   *
+   * @default - No services
+   */
+  readonly services?: IService[];
+}
+
+/**
+ * Attributes for importing an existing VPC Lattice service network
+ */
+export interface ServiceNetworkAttributes {
+  /**
+   * The ARN of the service network
+   */
+  readonly serviceNetworkArn: string;
+
+  // TODO: restore ID? Constructing the ARN from the ID will need accountId, region, etc
+  // readonly serviceNetworkId?: string;
```

Comment:
We talked about this on the call, it's a shorthand to for resources in the same account/region. Less powerful than Arn, but more portable.

Reply:
> it's a shorthand to for resources in the same account/region

This feels like an unwritten agreement of sorts - I'm not sure it is safe to rely on users intuitively understanding it.

Would it be overkill to actually accept the account ID/region and build the ARN using those?